requirements-freeze.txt

```diff
@@ -1,19 +1,18 @@
 Babel==2.3.4
 Beaker==1.9.0
-bleach==3.3.0
+bleach==3.1.4
```

Comment:
Any idea why it downgraded bleach?

Reply:
3.3.0 was manually added in the freeze file. the auto generated freeze file will overwrite it.
We need to take another look at https://github.com/GSA/inventory-app/pull/246/files
if bleach is latest version, then we dont need a pinned version in the requirements.txt any more.

Reply:
@FuhuXia This is the lasted version of bleach and could be set without it being pinned to 3.3.0. Although in the [CKAN master,](https://github.com/ckan/ckan/blob/master/requirements.txt#L10) bleach is pinned at 3.3.0.

Reply:
Rebased to inventory_ckan_2.8 for the requirements.txt change and this time it stays at 3.3.0. marking resolved.